spec/requests/api/medias_spec.rb

```diff
@@ -163,49 +163,6 @@
         end
       end
 
-      # response '429', 'API limit reached' do
```

Comment:
**question** can we just delete these?

Reply:
I deleted because I think this is testing behavior that no longer exists. I could totally be misunderstanding, but I think this error handling came from the Twitter gem.

So I just deleted this. I'm not sure if there is a way to substitute this?